pkg/preflights/host-preflight.yaml

```
@@ -291,3 +303,29 @@ spec:
               message: Kernel meets minimum version version of 3.10
           - fail:
               message: Kernel doesn't meet minimum version of 3.10
+    - http:
+        checkName: Can Access API
```

The way these print out during a failure, this name will make more sense
```suggestion
        checkName: API Access
```

---

Think this one got missed. I'll just commit it.